nevermined_metadata/app/assets.py

```diff
@@ -65,28 +67,86 @@ def get_metadata(did):
     try:
         asset_record = get_dao().get(did)
         metadata = _get_metadata(asset_record['service'])
-        return Response(_sanitize_record(metadata), 200, content_type='application/json')
+        return Response(
+            _sanitize_record(metadata), 200, content_type='application/json'
+        )
     except Exception as e:
         logger.error(e)
         return f'{did} asset DID is not in MetadataDB', 404
 
 
+@assets.route('/service/<serviceId>', methods=['GET'])
+def get_service(serviceId):
+    """Get the Service with a particular serviceId
+    swagger_from_file: docs/get_service.yml
+    """
+    try:
+        asset_record = get_dao().get_service(serviceId)
+        return Response(
+            _sanitize_record(asset_record), 200, content_type='application/json'
+        )
+    except Exception as e:
+        logger.error(e)
+        return f'{serviceId} service is not in the MetadataDB', 404
```

Comment:
I think we should change this to return a 404 if the record does not exist and a 500 in case there is a server error. This way things don't fail silently